odkx-src/services-managing.rst

```diff
@@ -61,6 +61,23 @@ Before you are able to synchronize data or application files to a device, you wi
 
       If you are using SSL, be sure to specify :code:`https://...`
 
+    .. _services-login-with-qr:
+
+    Additionally the app also provides an easy and quick option to login with QR code
+
+      1. Click on the QR icon at the top right corner.
```

Comment:
in the top right corner, not at. Top right corner of what?

Reply:
Since this section is part of Step 2 of Server Configuration and it is already mentioned in Step 1 to go to server settings under the settings menu. Should I mention this again?

Reply:
So I went and looked where this is in: https://docs.odk-x.org/services-managing/?highlight=ssl

There is no QR icon in these screen shots. If this is the right page, the images may need to be updated. Then if it's among images with the QR code icon, could just say "Click on the QR icon at the top right corner" and add an additional screenshot with that circled in red.

Reply:
Will it be "at the top right corner" this time?

Reply:
Should be "in top right corner", sorry, mentioned that earlier but then was just repeating back what was in the text.

Reply:
Okay thanks